azure-communication-ui/azure-communication-ui/build.gradle

```diff
@@ -82,7 +82,7 @@ dependencies {
     implementation "androidx.constraintlayout:constraintlayout:$constraint_layout"
     implementation 'androidx.navigation:navigation-fragment-ktx:2.3.5'
     implementation "com.azure.android:azure-core:$azure_core_version"
-    implementation "com.azure.android:azure-communication-calling:$calling_sdk_version"
+    api "com.azure.android:azure-communication-calling:$calling_sdk_version"
```

Comment:
Do we want this to be API? I'd think we'd want to hide this dependency downstream.

Reply:
Is there dependencies in the demo for this and can we strip them out?

Reply:
We need to expose a few types from the SDK related to Authentication

Reply:
Hmmm, ok makes sense.